src/js/update-sites.js

```diff
@@ -20,26 +22,29 @@ let vm = new Vue({
     packages: [],
     codes: [],
     sites: [],
-    siteQuery: []
+    siteQuery: '?where={"code.package":"588b90ab36447521c5af51e4"}'
   },
   methods: {
     getSites(siteQuery) {
       let that = this;
-      let endpoint = '/sites';
+      let endpoint = 'sites';
+      console.log(packageBaseURL+endpoint+siteQuery);
 
-      atlasRequest(baseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      atlasRequest(packageBaseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      setTimeout(remDup, 50);
     },
     getCodes() {
       let that = this;
-      let endpoint = '/code';
+      let endpoint = 'code';
+      console.log(packageBaseURL+endpoint);
 
-      atlasRequest(baseURL, endpoint, query = '', method = 'GET', body = null, etag = null).then(data => that.codes = data);
+      atlasRequest(packageBaseURL, endpoint, query = '', method = 'GET', body = null, etag = null).then(data => that.codes = data);
     },
     sendToAtlas() {
-      let siteOriginalPackages = this.sites._items;
+      let siteOriginalPackages = this.sites[0];
```

Comment:
What if there is `this.sites[1]`? If there are more than 500 sites returned in the query then another element in the array will exist. You can test this by looking for the current core or profile code asset.